modules/tensor_mechanics/tests/Wave_1_D/HHT_time_integration/wave_bc_1d.i

```diff
@@ -0,0 +1,346 @@
+  [../]
+  [./vel_y]
+    type = NewmarkVelAux
+    variable = vel_y
+    acceleration = accel_y
+    gamma = 0.6
+    execute_on = timestep_end
+  [../]
+  [./accel_z]
+    type = NewmarkAccelAux
+    variable = accel_z
+    displacement = disp_z
+    velocity = vel_z
+    beta = 0.3025
+    execute_on = timestep_end
+  [../]
+  [./vel_z]
+    type = NewmarkVelAux
+    variable = vel_z
+    acceleration = accel_z
+    gamma = 0.6
+    execute_on = timestep_end
+  [../]
+  [./stress_yy]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    variable = stress_yy
+    index_i = 0
+    index_j = 1
+  [../]
+  [./strain_yy]
+    type = RankTwoAux
+    rank_two_tensor = total_strain
+    variable = strain_yy
+    index_i = 0
+    index_j = 1
+  [../]
+
+[]
+
+
+[BCs]
+  [./top_y]
+    type = DirichletBC
+    variable = disp_y
+    boundary = top
+    value=0.0
+  [../]
+  [./top_x]
+    type = DirichletBC
+    variable = disp_x
+    boundary = top
+    value=0.0
+  [../]
+  [./top_z]
+    type = DirichletBC
+    variable = disp_z
+    boundary = top
+    value=0.0
+  [../]
+  [./right_x]
+    type = DirichletBC
+    variable = disp_x
+    boundary = right
+    value=0.0
+  [../]
+  [./right_z]
+    type = DirichletBC
+    variable = disp_z
+    boundary = right
+    value=0.0
+  [../]
+  [./left_x]
+    type = DirichletBC
+    variable = disp_x
+    boundary = left
+    value=0.0
+  [../]
+  [./left_z]
+    type = DirichletBC
+    variable = disp_z
+    boundary = left
+    value=0.0
+  [../]
+  [./front_x]
+    type = DirichletBC
+    variable = disp_x
+    boundary = front
+    value=0.0
+  [../]
+  [./front_z]
+    type = DirichletBC
+    variable = disp_z
+    boundary = front
+    value=0.0
+  [../]
+  [./back_x]
+    type = DirichletBC
+    variable = disp_x
+    boundary = back
+    value=0.0
+  [../]
+  [./back_z]
+    type = DirichletBC
+    variable = disp_z
+    boundary = back
+    value=0.0
+  [../]
+  [./bottom_x]
+    type = DirichletBC
+    variable = disp_x
+    boundary = bottom
+    value=0.0
+  [../]
+  [./bottom_z]
+    type = DirichletBC
+    variable = disp_z
+    boundary = bottom
+    value=0.0
+  [../]
+  [./bottom_y]
+    type = FunctionPresetBC
+    variable = disp_y
+    boundary = bottom
+    function = displacement_bc
+  [../]
+[]
+
+[Materials]
+  [./Elasticity_tensor]
+    type = ComputeElasticityTensor
+    block = 0
+    fill_method = symmetric_isotropic
+    C_ijkl = '1 0'
+  [../]
+
+  [./strain]
+    type = ComputeSmallStrain
+    block = 0
+    displacements = 'disp_x disp_y disp_z'
+  [../]
+
+  [./stress]
+    type = ComputeLinearElasticStress
+    store_stress_old = true
+    block = 0
+  [../]
+
+  [./density]
+    type = GenericConstantMaterial
+    block = 0
+    prop_names = 'density'
+    prop_values = '1'
+  [../]
+
+[]
+
+[Executioner]
+  type = Transient
+  start_time = 0
+  end_time = 6.0
+  dtmax = 0.1
+  dtmin = 0.1
+  l_tol = 1e-8
+  nl_rel_tol = 1e-8
+  [./TimeStepper]
+    type = ConstantDT
```

Comment:
For ConstantDT you could just omit the TimeStepper block and set dt at Executioner level...